### DISCUSSION TITLE
能出个看视频自动选择最高清晰度吗？

### DISCUSSION THREAD

**Discussion by hcl55** [the1812/Bilibili-Evolved#4470] [功能建议 / Ideas]
我充大会员之后，发现B站总是会自动调回高清画质，每次都要点击
![image](https://github.com/the1812/Bilibili-Evolved/assets/89288837/450506ba-dd21-4cdc-9fc0-bf410c7784c7)

**Comment by Geeyun-JY3** [comment]
#3898、#4357

**Reply by hcl55** [reply]
还是不行，可能是新版页面的播放器问题，会自动换到1080p 高清的，然后我开了旧版页面的脚本后，换了高清晰度能保持，但是关掉脚本后，新版页面的播放器又会自动切换，真给我恶心坏了

**Reply by Geeyun-JY3** [reply]
还是去上述讨论里跟评吧，没必要再开重复讨论。

**Reply by hcl55** [reply]
好的，我找到自动选择最高清晰度的脚本了，但我不知道怎么关闭这个讨论

**Reply by the1812** [reply]
随用随走吧, 你想关闭也可以点下面的 Close discussion
